btrack/btypes.py

```diff
@@ -82,30 +121,38 @@ def probability(self, probability):
         self._raw_probability = probability
 
     @property
-    def state(self):
+    def state(self) -> constants.States:
         return constants.States(self.label)
 
-    def to_dict(self):
+    def to_dict(self) -> Dict[str, Union[bool, int, float]]:
         """Return a dictionary of the fields and their values."""
         stats = {k: getattr(self, k) for k, _ in PyTrackObject._fields_}
         stats.update(self.properties)
         return stats
 
     @staticmethod
-    def from_dict(properties: dict):
+    def from_dict(
+        properties: Dict[str, Union[bool, int, float]]
+    ) -> "PyTrackObject":
```

Comment:
Can't the output just be `PyTrackObject`?

Reply:
i.e. no quotes

Reply:
I think it may end up with a circular reference?

Reply:
Sorry, hadn't seen this was a method of `PyTrackObject`.

Looks like in `python==3.11` what I was suggesting will be possible https://stackoverflow.com/a/70932112/7359333

For now we can either keep it as it is or use `from __future__ import annotations` https://stackoverflow.com/a/33533514/7359333

Reply:
I went with your last option here as that seems to be the way things are going 😄